gnd/src/androidTest/java/com/google/android/gnd/persistence/remote/FakeRemoteDataStore.java

```diff
@@ -69,11 +73,13 @@
     return Flowable.empty();
   }
 
+  @Nullable
   @Override
   public Single<ImmutableList<ValueOrError<Observation>>> loadObservations(Feature feature) {
     return null;
   }
 
+  @Nullable
```

Comment:
If `@Nullable` is the default, can we remove this annotation throughout for brevity? Or is it recommended regardless?